src/components/Header.tsx

```diff
@@ -29,7 +24,7 @@ export function Header({selectedTheme, onClick}: HeaderProps) {
 
   return (
     <div className="d-flex justify-content-center">
-      <img style={logoStyle} src={logo} onClick={onClick} alt="logo" />
+      <img className="cursor-pointer h-[250px]" src={logo} onClick={onClick} alt="logo" />
```

Comment:
Could do the cursor with existing bootsrap. https://getbootstrap.com/docs/4.5/content/reboot/#pointers-on-buttons

Not sure about pixel height though

Reply:
As far as I can tell you can only do percentages with bootstrap. Can't specify pixels

Reply:
yes, looks like it